tests/integration/tier0/removed-certificate/test_removed_certificate.py

```diff
@@ -0,0 +1,24 @@
+# TODO Test will be modified/removed after https://issues.redhat.com/browse/RHELC-762 gets resolved
+import os.path
+
+import pytest
+
+
+@pytest.mark.removed_certificate
+def test_removed_certificate(convert2rhel, shell):
+    """
+    Verify, that the redhat-uep.pem certificate is installed if it is needed.
+    """
+    assert shell("rpm -qi subscription-manager").returncode == 0
+    assert (
+        shell(
+            "curl -o /etc/yum.repos.d/convert2rhel.repo https://ftp.redhat.com/redhat/convert2rhel/8/convert2rhel.repo"
+        ).returncode
+        == 0
+    )
+    assert os.path.exists("/etc/yum.repos.d/convert2rhel.repo") is True
+
+    for run in range(2):
+        with convert2rhel("-y --username username") as c2r:
+            assert c2r.expect("Password") == 0
+            c2r.sendcontrol("c")
```

Comment:
If you stop the conversion let's say at `TASK - [Convert: Subscription Manager - Enable RHEL repositories]` which follows the removal of the `centos-linux-release` package, the second conversion stops with a traceback:
```
[11/03/2022 15:28:29] TASK - [Prepare: Checking if the installed packages are up-to-date] *******
[11/03/2022 15:28:29] DEBUG - Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/convert2rhel/main.py", line 100, in main
    checks.perform_system_checks()
  File "/usr/lib/python3.6/site-packages/convert2rhel/checks.py", line 126, in perform_system_checks
    check_package_updates()
  File "/usr/lib/python3.6/site-packages/convert2rhel/checks.py", line 656, in check_package_updates
    packages_to_update = get_total_packages_to_update(reposdir=reposdir)
  File "/usr/lib/python3.6/site-packages/convert2rhel/pkghandler.py", line 1037, in get_total_packages_to_update
    packages = _get_packages_to_update_dnf(reposdir=reposdir)
  File "/usr/lib/python3.6/site-packages/convert2rhel/pkghandler.py", line 1079, in _get_packages_to_update_dnf
    base.read_all_repos()
  File "/usr/lib/python3.6/site-packages/dnf/base.py", line 545, in read_all_repos
    for repo in reader:
  File "/usr/lib/python3.6/site-packages/dnf/conf/read.py", line 42, in __iter__
    for r in self._get_repos(self.conf.config_file_path):
  File "/usr/lib/python3.6/site-packages/dnf/conf/read.py", line 109, in _get_repos
    parser.setSubstitutions(substs)
  File "/usr/lib64/python3.6/site-packages/libdnf/conf.py", line 1980, in setSubstitutions
    return _conf.ConfigParser_setSubstitutions(self, substitutions)
TypeError: in method 'ConfigParser_setSubstitutions', argument 2 of type 'std::map< std::string,std::string,std::less< std::string >,std::allocator< std::pair< std::string const,std::string > > > const &'
```

I've described it under https://issues.redhat.com/browse/RHELC-762.
The workaround of copying the CA cert is not enough. We need to solve the root cause - failing rollback. More specifically, solve the failing re-installation of the backed up packages.